staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -88,7 +88,7 @@ func (s *EtcdOptions) AddFlags(fs *pflag.FlagSet) {
 		"Enable watch caching in the apiserver")
 
 	fs.StringVar(&s.StorageConfig.Type, "storage-backend", s.StorageConfig.Type,
-		"The storage backend for persistence. Options: 'etcd3' (default), 'etcd2'.")
+		"The storage backend for persistence. Options: 'etcd2', 'etcd3'.")
```

Comment:
ah, I see there is `Validate` in L61 of this file. How about validate the flag there?